lib/ethereum-light-client-types/src/light_client_update.rs

```diff
@@ -15,11 +15,6 @@ pub enum LightClientUpdate {
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "bincode", derive(bincode::Encode, bincode::Decode))]
 pub struct EpochChangeUpdate {
```

Comment:
we don't need two separate types anymore right?

Reply:
I'm still in favor of having the enum and such instead of optional sync committee's it makes the code messier and harder to understand.

Reply:
ah, didn't realize there were still more fields in this struct. 👍